test/membrane_opus/encoder_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule Membrane.Opus.Encoder.EncoderTest do
+  use ExUnit.Case, async: true
+
+  alias Membrane.Opus.Encoder
+  alias Membrane.Opus.Support.Reader
+  alias Membrane.Caps.Audio.Raw
+  import Membrane.Testing.Assertions
+  alias Membrane.Testing
+  alias Membrane.Element.File
+  import Membrane.ParentSpec
+
+  @in_path "input.wav"
```

Comment:
What do you think about using raw packets from `fixtures` in the same way decoder tests use them instead of `wav` file? We probably could take raw pacets, encode them end compare with encoded version that is in `fixtures` or decode them and check if they are the same as before encoding? Or please include some example `wav` file in repo for tests.

Reply:
i don't think that's possible unless `Testing.Source` gets a `handle_demand` that can accept bytes :(